public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module

```diff
@@ -165,13 +166,14 @@ function os2web_accessibility_theme() {
       'variables' => array(),
     ),
     'os2web_accessibility' => array(
-      'variables' => array(
+      'arguments' => array(
```

Comment:
why did we need to rename it?
`variables` is default key for the theme array.

Reply:
@stanbellcom I revert these changes, it was my fault